scripts/make-dist.sh

```diff
@@ -4,23 +4,18 @@
 # in the LICENSE file.
 set -e
 
-package_name=singularity
-package_version=`(git describe --match 'v[0-9]*' --always 2>/dev/null || cat VERSION 2>/dev/null || echo "") | sed -e "s/^v//;s/-/_/g;s/_/-/;s/_/./g"`
-package_version_short=`(git describe --abbrev=0 --match 'v[0-9]*' --always 2>/dev/null || cat VERSION 2>/dev/null || echo "") | sed -e "s/^v//;s/-/_/g;s/_/-/;s/_/./g"`
+# spec file needs to be at the root of the project
```

Comment:
If spec file needs to be at the root of the project, why is it generated in the dist/rpm folder? I recommend that we generate it at the root of the project via mconfig right away and remove these 2 lines here.